.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 ci:
   autoupdate_schedule: "quarterly"
-files: ^openff|(^examples/((?!deprecated).)*$)|^docs
+files: ^openff|(^examples/((?!deprecated).)*$)|^docs|(^utilities/((?!deprecated).)*$)
```

Comment:
This (and a similar change below) are pretty much the only non-automated changes here